cmd/controller/main.go

```diff
@@ -69,7 +70,8 @@ func main() {
 	flag.BoolVar(&enableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
 	opts := zap.Options{
-		Development: true,
+		TimeEncoder: zapcore.RFC3339NanoTimeEncoder,
+		ZapOpts:     []zaplog.Option{zaplog.AddCaller()},
 	}
 	opts.BindFlags(flag.CommandLine)
```

Comment:
Can you add https://github.com/kubernetes-sigs/kueue/blob/a7681277ea932aef0b233e635447c0e16b31e116/cmd/kueue/main.go#L118 something like that here?

I think this would not allow us to configure the logger without this change.

Reply:
Sorry you mean to add what at line 79 or something else?

Reply:
Yes, 

```
zap.New(zap.UseFlagOptions(&opts)))
```

This should have use the same zap configuration for other controller runtime features and not require us to specify custom logger controls.

Reply:
@kannon92 Isn’t this initialization already in the code here : https://github.com/openshift/instaslice-operator/blob/f526dbcb416a13cb079b55e2fa7cf5f1848d6aad/cmd/controller/main.go#L79

Reply:
Yes it is..